config/samples/ramendr_v1alpha1_drpolicy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: ramendr.openshift.io/v1alpha1
```

Comment:
This file is not required.

Reply:
Yeah I think this got in during testing. Removing.